### DISCUSSION TITLE
Suggestions for compatible vacuum

### DISCUSSION THREAD

**Discussion by ingoau** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#622] [General]
Hi all

I'm looking to setup a robot vacuum.
I came across this card a while ago and thought it was pretty cool.
I do not have a robot vacuum yet and I am looking for recommendations that are compatible with this card.

Here are my requirements:

* Good for pet hair (rabbits)
* Compatible with all of the features of this card (or most)
* Preferably with a emptying station

Thanks
Inglan

**Comment by PiotrMachowski** [comment]
https://www.youtube.com/@VacuumWars

I think the most compatible are Roborock and Dreame vacuums. 

It is also worth to consider [Valetudo](https://valetudo.cloud/)

**Comment by ingoau** [comment]
Would the Roborock Q5+ be compatible?

**Reply by PiotrMachowski** [reply]
To be honest, you probably look at it in the wrong order. This card just displays data retrieved from the vacuum by different integrations, so you should find an integration and check if the vacuum is supported by it. From the card's perspective vacuum model doesn't matter, only integration matters.

Having said that, it probably is compatible using [custom Roborock integration](https://github.com/humbertogontijo/homeassistant-roborock)
